docs/rfc/ethereum_attestation.md

```diff
@@ -0,0 +1,126 @@
+# RFC: Ethereum attestation
+
+* Author: @CodeSandwich
+* Date: 2021-03-05
+* Status: draft
+* Community discussion: n/a
+
+## Motivation
+
+The attestation between Link and Ethereum is a valuable building block for a user identity.
+It brings the Link reputation coming from projects and contributions to the
+Ethereum world of DAOs and donations, where it's important to know, who's behind an address.
+On the other hand it lends Ethereum account reputation with its assets and undeniable history
+to Link to build user's trust in the identity.
+
+## Overview
+
+This RFC is built on top of [Identity Proofs RFC][rfc].
+It introduces support for Ethereum address claims on Link
+and a smart contract on Ethereum to make Link identity claims on Ethereum.
+
+## Link identity JSON extension
+
+The identity JSON supports a new key: `ethereum`.
+Under this key there is stored an ethereum address claim following this convention:
+
+- `account` - the claimed ethereum address, encoded according to [EIP-55][eip-55],
+e.g. using [ethers.js][ethers-addr]
+- `proof` - always `null`
```

Comment:
This is optional, so if we aren't using it, it shouldn't be set?

Reply:
Fixed